pkg/template/template_test.go

```diff
@@ -116,7 +116,7 @@ func TestParameterGenerators(t *testing.T) {
 	for i, test := range tests {
 		processor := NewProcessor(test.generators)
 		template := api.Template{Parameters: []api.Parameter{test.parameter}}
-		err, _ := processor.GenerateParameterValues(&template)
+		err := processor.GenerateParameterValues(&template)
```

Comment:
Would like to see tests verifying the field path on the returned error is correct, since clients will need to use those to correlate errors